source/app/models/url.rb

```diff
@@ -0,0 +1,32 @@
+require 'uri'
+
+class Url < ActiveRecord::Base
+
+attr_accessor :click_count
```

Comment:
Since ActiveRecord defines getter and setter methods for the columns in the DB, you shouldn't define them too.

Reply:
Got it, thanks @mikegee!